packages/components/src/summary-button/style.scss

```diff
@@ -73,4 +73,8 @@
 			@include body-medium;
 		}
 	}
+
+	&:focus {
+		box-shadow: 0 0 0 1.5px var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9));
```

Comment:
Code looks right!

One consideration, in core we have a mixin file for these styles, can you import one and apply that here?

https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_mixins.scss#L131